### ISSUE TITLE
Consumption of cold delicatessen meat

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9943]
**Preferred term label:**
Consumption of cold delicatessen meat

**Synonyms**
Cold cuts, cold hot dogs

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/books/NBK534838/ "Transmission of the bacteria occurs via the fecal-oral route and most commonly involves foods such as cold deli meats"

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Listeriosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
this probably has an extremely low positive predictive value. (thankfully), and I am guessing it is only useful after the diagnosis has been made?

**Comment by MickeySegal** [comment]
We've changed our finding name to "Cold deli meat may have become contaminated".
https://www.ncbi.nlm.nih.gov/books/NBK534838/
"Transmission of the bacteria occurs via the fecal-oral route and most commonly involves foods such as cold deli meats and unpasteurized dairy products...Foods that have the highest rates of L. monocytogenes related infections include:

Raw sprouts
Unpasteurized milk
Soft cheeses
Cold deli meats
Cold hot dogs
Smoked seafood"

If you are expecting the diagnosis, this is a useful finding about which to ask.  Since adding this I've become a lot more careful about how long sliced meat is allowed to stay in the refrigerator.

**Comment by pnrobinson** [comment]
I think it is probably better to consider making terms in ECTO (https://www.ebi.ac.uk/ols4/ontologies/ecto) @mellybelly @cmungall 
There are so many possibly ways of getting Listeria, and this differs from country to country (e.g., raw pork is apparently a risk factor in China:PMID: [29896170](https://pubmed.ncbi.nlm.nih.gov/29896170)). 
The scope of HPO does not include all possible exposures, which would be considerable.
ECTO has a broad scope and we would be interested in feedback as to whether it is useful for this sort of thing.
